python/geotag_from_gpx.py

```diff
@@ -109,7 +109,16 @@ def exif_time(filename):
     '''
     m = pyexiv2.ImageMetadata(filename)
     m.read()
-    return m['Exif.Photo.DateTimeOriginal'].value
+    image_millisec = 0
```

Comment:
@Nakaner Good find with the missing support of sub seconds. We've also tried to solve this https://github.com/mapillary/mapillary_tools/blob/master/python/lib/exif.py#L166. Would you like to help replace this part with `extract_capture_time`?